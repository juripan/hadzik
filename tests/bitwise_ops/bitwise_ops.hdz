naj x = 100 & 12 // 4
naj y = 30 | 17 // 31
naj z = x ^ y // 27

naj bin_neg = ~100 + 1 // -100
naj neg = -100

kec bin_neg == neg {
    hutor("equal")
}

vychod(z)