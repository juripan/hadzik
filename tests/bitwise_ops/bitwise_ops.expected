stdout: equal| stderr: | returncode: 27